pkg/util/BUILD

```diff
@@ -64,3 +64,9 @@ filegroup(
     ],
     tags = ["automanaged"],
 )
+
+sh_test(
+    name = "verify-util-pkg",
+    srcs = ["//hack:verify-util-pkg.sh"],
+    data = glob(["*.go"]),
```

Comment:
Can you add: `size = "small",` after this? Getting the following warning when testing locally:
```
WARNING: //pkg/util:verify-util-pkg: Test execution time (0.1s excluding execution overhead) outside of range for MODERATE tests. Consider setting timeout="short" or size="small".
```